### PR TITLE
Update ansible extension information

### PR DESCRIPTION
**Pull request by ganeshrn**
*  Update ansible extesnion information
   for vscode, vim, emacs
*  Update ansible-language-server information
   in other tools section

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/community/other_tools_and_programs.rst [hunk 1]
```diff
@@ -58,17 +59,16 @@ Visual studio code
 
 An open-source, free GUI text editor created and maintained by Microsoft. Useful Visual Studio Code plugins include:
 
-
+* `Ansible support by Red Hat <https://marketplace.visualstudio.com/items?itemName=redhat.ansible>`_ - provides autocompletion, syntax highlighting, hover, diagnostics, goto support and command to run ansible-playbook and ansible-navigator tool for both local and execution-environment setups.
```
Comment (Akasurde) [inline docs/docsite/rst/community/other_tools_and_programs.rst]:
```suggestion
* `Ansible support by Red Hat <https://marketplace.visualstudio.com/items?itemName=redhat.ansible>`_ - provides autocompletion, syntax highlighting, hover, diagnostics, goto support, and command to run ansible-playbook and ansible-navigator tool for both local and execution-environment setups.
```

File: docs/docsite/rst/community/other_tools_and_programs.rst [hunk 2]
```diff
@@ -58,17 +59,16 @@ Visual studio code
...
-* `Ansible Syntax Highlighting Extension <https://marketplace.visualstudio.com/items?itemName=haaaad.ansible>`_ - YAML & Jinja2 support.
-* `Visual Studio Code extension for Ansible <https://marketplace.visualstudio.com/items?itemName=vscoss.vscode-ansible>`_ - provides autocompletion, syntax highlighting.
 
 vim
 ===
 
 An open-source, free command-line text editor. Useful vim plugins include:
 
 * `Ansible vim <https://github.com/pearofducks/ansible-vim>`_  - vim syntax plugin for Ansible 2.x, it supports YAML playbooks, Jinja2 templates, and Ansible's hosts files.
+* `Ansible vim and neovim plugin <https://www.npmjs.com/package/@yaegassy/coc-ansible>`_  - vim plugin (lsp client) for Ansible, it supports autocompletion, syntax highlighting, hover, diagnostics and goto support.
```
Comment (Akasurde) [inline docs/docsite/rst/community/other_tools_and_programs.rst]:
```suggestion
* `Ansible vim and neovim plugin <https://www.npmjs.com/package/@yaegassy/coc-ansible>`_  - vim plugin (lsp client) for Ansible, it supports autocompletion, syntax highlighting, hover, diagnostics, and goto support.
```

File: docs/docsite/rst/community/other_tools_and_programs.rst (hunk 1 above)
Comment (samccann) [inline docs/docsite/rst/community/other_tools_and_programs.rst]: Why is the title "Ansible Support" ? Shouldn't be Ansible VS Code extension?

**Comment by samccann** [comment]
This PR makes almost the same changes as https://github.com/ansible/ansible/pull/77234. Can you work with @ssbarnea to determine which is correct?

File: docs/docsite/rst/community/other_tools_and_programs.rst (hunk 1 above)
Comment (samccann) [inline docs/docsite/rst/community/other_tools_and_programs.rst]:
```suggestion
* `Ansible extension by Red Hat <https://marketplace.visualstudio.com/items?itemName=redhat.ansible>`_ - provides autocompletion, syntax highlighting, hover, diagnostics, goto support and command to run ansible-playbook and ansible-navigator tool for both local and execution-environment setups.
```

File: docs/docsite/rst/community/other_tools_and_programs.rst [hunk 3]
```diff
@@ -58,17 +59,16 @@ Visual studio code
 
 An open-source, free GUI text editor created and maintained by Microsoft. Useful Visual Studio Code plugins include:
 
-
+* `Ansible support by Red Hat <https://marketplace.visualstudio.com/items?itemName=redhat.ansible>`_ - provides autocompletion, syntax highlighting, hover, diagnostics, goto support, and command to run ansible-playbook and ansible-navigator tool for both local and execution-environment setups.
```
Comment (samccann) [inline docs/docsite/rst/community/other_tools_and_programs.rst]:
```suggestion
* `Ansible extension by Red Hat <https://marketplace.visualstudio.com/items?itemName=redhat.ansible>`_ - provides autocompletion, syntax highlighting, hover, diagnostics, goto support, and command to run ansible-playbook and ansible-navigator tool for both local and execution-environment setups.
```

**Review by samccann — APPROVED** [review]

**Pull request merged**